pymc/tests/test_model.py

```diff
@@ -702,6 +702,13 @@ def test_nested_model_coords():
     assert set(m2.RV_dims) < set(m1.RV_dims)
 
 
+def test_add_coord_fixed_dims():
+    with pm.Model(coords={"feature": [1], "group": ["A", "B"]}):
```

Comment:
Add a comment to this one since its not obvious from reading how fixed dims are being tested